binance-dex/package.json

```diff
@@ -1,15 +1,43 @@
 {
   "name": "@chainlink/binance-dex-adapter",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "license": "MIT",
-  "main": "index.js",
+  "keywords": [
+    "Chainlink",
+    "LINK",
+    "blockchain",
+    "oracle"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "url": "https://github.com/smartcontractkit/external-adapters-js",
+    "type": "git"
+  },
   "scripts": {
-    "server": "node -e 'require(\"./index.js\").server()'",
```

Comment:
We need to keep this `server` script, as it is used in Docker runs. Please add it accordingly in all adapters.